Gemfile

```diff
@@ -44,6 +44,14 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  # Brakeman is used to detect security risks in the application
+  gem 'brakeman', :require => false
+  # RuboCop is a static code analyzer tool that will ensure code quality
+  gem 'rubocop', require: false
+  # Looks for query optimations to avoid n+1 issues. Read more: https://sitepoint.com/silver-bullet-n1-problem
+  #gem 'bullet' # Has an issue with no support mongoid v6.0.0, re-asses later
```

Comment:
Could you change this to:

    # TODO: Has an issue with no support mongoid v6.0.0, re-asses later
    #gem 'bullet'

To keep with the current TODO style?